src/client/pages/ConnectPayment/components/TrustlyModal.tsx

```diff
@@ -94,11 +94,16 @@ export const TrustlyModal: React.FC<Props> = ({
           ref={iframeRef}
           onLoad={async () => {
             const contentWindow = iframeRef.current?.contentWindow
-            await actualHandleIframeLoad(
-              setIsOpen,
-              setIsSuccess,
-              generateTrustlyUrl,
-            )(contentWindow)
+
+            try {
+              await actualHandleIframeLoad(
+                setIsOpen,
+                setIsSuccess,
+                generateTrustlyUrl,
+              )(contentWindow)
+            } catch (e) {
+              console.error(e.message)
```

Comment:
i guess we could a) use just `console.error(e)` right?, and b) won't sentry capture this anyways if we write it on `console.error`? maybe thats ok and we can ignore it in sentry instead

Reply:
That's what I wanted to test Sentry locally for ;) . I did not receive and issue when catching the error. (I spammed that connect button like a pro). 
 But if I removed the try/catch i got an issue in Sentry instantly.

We can use console.error(e to  get the whole stacktrace as well but I though the idea was just using this as a reminder and if you just use console.error(e) the output is a bit messy.